modules/lang-painless/src/main/antlr/PainlessParser.g4

```diff
@@ -156,11 +156,11 @@ postdot
     ;
 
 callinvoke
-    : DOT DOTID arguments
+    : nullSafe=COND? DOT DOTID arguments
```

Comment:
I did it this way to make it clear that it was a thing I was going to check and because it looks like `COND()` would invoke `getToken` which looks surprisingly non-trivial. I figured this way antlr saves a copy for me and I can just check it.